build-tools/packages/build-tools/src/repoPolicyCheck/handlers/fluidBuildTasks.ts

```diff
@@ -467,23 +550,36 @@ export const handlers: Handler[] = [
 			const ignore = getFluidBuildTasksTscIgnore(root);
 			for (const script in json.scripts) {
 				// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-				const command = json.scripts[script]!;
-				if (shouldProcessScriptForTsc(script, command, ignore)) {
-					try {
-						const checkDeps = getTscCommandDependencies(
-							packageDir,
-							json,
-							script,
-							command,
-							deps,
-						);
-						// Check the dependencies
-						const error = checkTaskDeps(root, json, script, checkDeps);
-						if (error) {
-							errors.push(error);
+				const commands = json.scripts[script]!;
+				for (const commandUntrimmed of commands.split("&&")) {
+					const command = commandUntrimmed.trim();
+					if (shouldProcessScriptForTsc(script, command, ignore)) {
+						try {
+							const checkDeps = getTscCommandDependencies(
+								packageDir,
+								json,
+								script,
+								command,
+								deps,
+							);
+							// Check the dependencies
+							const error = checkTaskDeps(root, json, script, checkDeps);
+							if (error) {
+								// eslint-disable-next-line no-debugger
+								debugger;
```

Comment:
Did you intend to remove the 'debugger;' statement?  (ditto below).

Reply:
Thank you. I had removed them, then added back without a plan to commit more changes, then I apparently committed another change. Removing...